style.css

```diff
@@ -1,9 +1,3 @@
-@media screen and (min-width: 768px) {
-  #container {
-    height: 100%;
-    justify-content: center;
-  }
-}
 
 main {
```

Comment:
- [OPTIONAL] You can add the CSS smooth scroll to your CSS file so that when a user clicks on one of the navigation links, the website scrolls smoothly to that section. It will greatly improve the user experience of  your portfolio: 😄 
> You can achieve this by adding a line of CSS to your project `html {
  scroll-behavior: smooth;
}`